src/microbe_stage/Microbe.cs

```diff
@@ -2108,6 +2108,75 @@ private Transform GetNewPhysicsRotation(Transform transform)
         return new Transform(new Basis(slerped), transform.origin);
     }
 
+    /// <summary>
+    ///   Adds torque to reach the targeted rotation
+    /// </summary>
+    private void ApplyPhysicsRotation(PhysicsDirectBodyState physicsState)
+    {
+        var transform = physicsState.Transform;
+        var target = transform.LookingAt(LookAtPoint, new Vector3(0, 1, 0));
+        if (transform == target)
+            return;
+
+        // For now strength uses only base strength
+        // Can later be affected by cilia
+        var strength = BaseRotationalStrength();
+
+        // Calculate the angle to reach the target rotation in degrees
+        var angleToTarget = target.basis.GetEuler().y - transform.basis.GetEuler().y;
+        angleToTarget = (float)(angleToTarget * (180.0f / Math.PI));
+        if (angleToTarget > 180.0f)
+        {
+            angleToTarget = -360.0f + angleToTarget;
+        }
+        else if (angleToTarget < -180.0f)
+        {
+            angleToTarget = 360.0f + angleToTarget;
+        }
+
+        // Maximum desired angular velocity
+        // If the value is to high the microbe will not brake in time
+        // If it is to low the microbe is rotating slower than it could
+        var maxVelocity = 1.0f + (strength * 0.3f) / (HexCount * 40.0f);
+
+        // Angular velocity of the microbe in a range from -90(-maxVelocity) to 90(maxVelocity)
+        var velocityTo90 = physicsState.AngularVelocity.y / maxVelocity * 90.0f;
+
+        // Apply torque
+        var torque = new Vector3(0, strength * RotationalForce(velocityTo90, angleToTarget) * physicsState.Step, 0);
+        physicsState.AddTorque(torque);
+    }
+
+    private float BaseRotationalStrength()
+    {
+        // More strength is required to rotate larger microbes
+        // Value should be lowered if cilia is implemented
+        // Uses CELL_BASE_THRUST for now, the same as for movement
+        var strength = Constants.CELL_BASE_THRUST * (1.0f + HexCount);
+
+        // Currently uses same modifiers as for movement
+        return strength * (Species.MembraneType.MovementFactor - (Species.MembraneRigidity *
+            Constants.MEMBRANE_RIGIDITY_MOBILITY_MODIFIER));
+    }
+
+    private float RotationalForce(float velocityTo90, float angleToTarget)
+    {
+        // Do nothing if velocity is nearly zero and rotation is nearly correct
+        if (Math.Abs(velocityTo90) < 0.1f && Math.Abs(angleToTarget) < 1.0f)
+        {
+            return 0.0f;
+        }
+
+        // Accelerate if rotating to slow
+        if (velocityTo90 / angleToTarget < 1.0f)
+        {
+            return Mathf.Sign(angleToTarget) * 1.0f;
```

Comment:
Is there a reason for the force to be 1, 0 or -1? This seems more of a directional thing here (ternary choice between increase, decrease and do nothing). If yes, you might want to rename the function.
Else, it would be better to have this base force of 1.0f as a constant in the dedicated file.